### ISSUE TITLE
App doesn't start, missing dependencies

### ISSUE THREAD

**Issue by maxgerhardt** [hongquan/CoBang#33]
Installing the software per [this](https://askubuntu.com/a/1262902/868804)

gives

```
$ cobang 
Traceback (most recent call last):
  File "/usr/bin/cobang", line 11, in <module>
    load_entry_point('cobang==0.5.6', 'console_scripts', 'cobang')()
  File "/usr/local/lib/python3.8/dist-packages/pkg_resources/__init__.py", line 488, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.8/dist-packages/pkg_resources/__init__.py", line 2861, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.8/dist-packages/pkg_resources/__init__.py", line 2461, in load
    return self.resolve()
  File "/usr/local/lib/python3.8/dist-packages/pkg_resources/__init__.py", line 2467, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3/dist-packages/cobang/__main__.py", line 17, in <module>
    from .app import CoBangApplication
  File "/usr/lib/python3/dist-packages/cobang/app.py", line 34, in <module>
    gi.require_version('Rsvg', '2.0')
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Rsvg not available
```

thus making it unstartable.

Then after finding [this](https://stackoverflow.com/a/19744099/5296568) and doing 

```
sudo apt-get install gir1.2-rsvg-2.0 python3-cairo python-gi-cairo python3-gi
```
the app was startable.

It seems you're missing some dependencies in the PPA install?

----

Extra info

```
$ lsb_release -a
#No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

```
$ python3 -V
Python 3.8.5
```

**Comment by hongquan** [comment]
Thank you @maxgerhardt for reporting this. It is true that I listed not enough dependencies.

**Comment by hongquan** [comment]
Fixed in v0.8.1.
